services/QuillLMS/spec/models/activity_session_feedback_history_spec.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: activity_session_feedback_histories
+#
+#  id                           :integer          not null, primary key
+#  activity_session_uid         :string
+#  feedback_history_session_uid :string
+#
+# Indexes
+#
+#  index_activity_sess_fb_histories_on_activity_session_uid  (activity_session_uid) UNIQUE
+#  index_activity_sess_fb_histories_on_feedback_history_uid  (feedback_history_session_uid) UNIQUE
+#
+require 'rails_helper'
+
+# it { shoulda cheatsheet: https://github.com/thoughtbot/it { shoulda-matchers#activemodel-matchers
+RSpec.describe ActivitySessionFeedbackHistory, type: :model do
+
+  context 'associations' do
+    it { should have_one(:activity_session) }
+    it { should have_many(:feedback_history) }
+  end
+
+  context 'validations' do
+    it { should validate_presence_of(:activity_session_uid) }
+    it { should validate_uniqueness_of(:activity_session_uid) }
+    it { should validate_presence_of(:feedback_history_session_uid) }
+    it { should validate_uniqueness_of(:feedback_history_session_uid) }
+  end
+
+  context '#self.get_feedback_history_session_uid' do
+    it 'should create a new record if the activity_session_uid is new' do
+      assert_equal ActivitySessionFeedbackHistory.count, 0
```

Comment:
So, these are `rspec` tests but they use `minitest` functions instead of the `rspec` ones, I'm not sure how we feel about this. We should probably stick with rspec here.
e.g. 
```ruby
# minitest
assert_equal ActivitySessionFeedbackHistory.count, 0
# rspec 
expect(ActivitySessionFeedbackHistory.count).to_be eq(0)